graph/telemetry/istio/appender/service_entry.go

```diff
@@ -116,7 +116,7 @@ func (a ServiceEntryAppender) applyServiceEntries(trafficMap graph.TrafficMap, g
 
 	// Replace "se-service" nodes with an "se-aggregate" serviceEntry node
 	for se, seServiceNodes := range seMap {
-		serviceEntryNode := graph.NewNode(namespaceInfo.Namespace, se.name, "", "", "", "", a.GraphType)
+		serviceEntryNode := graph.NewNode(graph.Unknown, namespaceInfo.Namespace, se.name, "", "", "", "", a.GraphType)
```

Comment:
As in unused_node appender, perhaps this should be the id of the home cluster?

Reply:
+1, something to look at.

Reply:
Added as subtask to https://github.com/kiali/kiali/issues/1833